print-workspace-status.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+git_commit="$(git describe --tags --always --dirty)"
+build_date="$(date -u '+%Y%m%d')"
+docker_tag="v${build_date}-${git_commit}"
+cat <<EOF
+STABLE_REGISTRY gcr.io
```

Comment:
I kinda see why you put these here, though this feels a little weird. I wish bazel had `vardef` so that we could have defaults in the BUILD file that could be overridden if desired.

Instead, maybe let these be overridden by environment variables here? e.g.
```bash
STABLE_REGISTRY ${REGISTRY:-gcr.io}
```
or something?